ocsci/__init__.py

```diff
@@ -6,3 +6,32 @@
 https://docs.pytest.org/en/latest/reference.html
 under section PYTEST_DONT_REWRITE
 """
+from dataclasses import dataclass, field, fields
```

Comment:
Can you add some header docs please on how this is getting used from high level

Reply:
@vasukulkarni added!